source/unix/uv_unix_fs.c

```diff
@@ -77,9 +77,11 @@
 
 #define PATH                                                                  \
   do {                                                                        \
-    (req)->path = strdup(path);                                               \
-    if ((req)->path == NULL)                                                  \
-      return -ENOMEM;                                                         \
+    if (path) {                                                               \
```

Comment:
Thank you for your comment. I will follow the coding style. I check the coding style guide in iotjs [https://github.com/Samsung/iotjs/wiki/Coding_Style_Guideline]. But I failed to find in the link. Could you let me know the link to the rule of explicitly writing the NULL in the condition? I want to read the rules or link, probably what I missed to read.